Rollbar-Projects-AccessToken/inputs/inputs_1_create.json

```diff
@@ -0,0 +1,10 @@
+{
+  "ProjectId": 581379,
```

Comment:
Could we parameterize this also. Instead of commenting on the rest of the inputs could we look through them and just parameterize any of the ids that will be different between publishers.